internal/relic/planar/pangalactic/pangalactic.go

```diff
@@ -0,0 +1,51 @@
+package pangalactic
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/relic"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+)
+
+const (
+	mod = key.Modifier("pan-galactic-commercial-enterprise")
```

Comment:
this should be `mod key.Modifier = "pan-galactic-commercial-enterprise"` (and yeah, fix that on other relics I totally missed that in other PR...)